common/bookmarks/wc_bookmarks.txt

```diff
@@ -0,0 +1,154 @@
+﻿# ENABLING THIS TEMPORARILY
```

Comment:
Too many bookmarks. And you can't scroll them.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/134036832-fef273c4-a5dc-4fda-a1f3-519848028388.png)

</details>

Reply:
> 
> 
> Too many bookmarks. And you can't scroll them.
> Click to expand

Fixed in 5227024ef949fd977c620a4bbcce083dc30deb31. Removed _Cata_ and _Legion_ bookmarks and added back the 583-605 bookmarks. There are still a lot of them, but they should all fit at the top properly now for testing purposes.